diff.sh

```diff
@@ -245,15 +242,18 @@ def process(lines, options):
         # Replace tabs with spaces
         diff_rows.append(row)
         originals.append(original)
+        line_nums.append(line_num)
+        if options.stop_jrra and mnemonic == 'jr' and row.split('\t')[1].strip() == 'ra':
```

Comment:
Functions could contain more than one `jr ra`... but I guess when that happens it's easy to turn this behavior off, and such functions are typically small since they don't contain any register cleanup, and small functions are less often diffed. The alternative would be to have the `--stop-jr-ra` flag take an optional count parameter, but again, I'm perfectly fine with being lazy here and keeping this as is.

Reply:
Fair. I'll leave this off by default then. That way its there if we need it but avoids cutting off the function.